### DISCUSSION TITLE
Wrong version Webpack

### DISCUSSION THREAD

**Discussion by hd321kbps** [webpack/webpack#15483] [Q&A]
![image](https://user-images.githubusercontent.com/9886849/157019419-3697f984-480f-46e6-b1e4-548c69b26d24.png)

**Other relevant information:**
webpack version:  5.70.0
Node.js version: 17.6.0
Operating System:  Windows 11

**Comment by vankop** [comment]
hm.. works fine for me. maybe you have` webpack` 5.65 installed globally?

**Reply by hd321kbps** [reply]
Install global version 5.70.0

**Reply by vankop** [reply]
then `webpack` in `PATH` (`$> which webpack`) points to another directory.. also `webpack` starts `webpack-cli`, so maybe `webpack-cli` resolves wrong webpack..

**Reply by hd321kbps** [reply]
![image](https://user-images.githubusercontent.com/9886849/157021943-0151e58f-350b-40b7-a10b-79f046216467.png)

![image](https://user-images.githubusercontent.com/9886849/157020417-e0675f79-bd0f-478e-befd-4b5de66e6223.png)

**Reply by vankop** [reply]
so global `webpack` points to `roaming/npm/webpack`. `npm view` points to `roaming/npm/node_modules/webpack`

**Reply by hd321kbps** [reply]
No, the first one points to a startup script.
Removed webpack and webpack-cli. Put on a new one, everything works. Webpack version 5.70.0 and webpack-cli 4.9.2.
Apparently there was some kind of desynchronization between them.

**Reply by hd321kbps** [reply]
Or do they conflict when both are set

**Reply by vankop** [reply]
I don't understand what you mean.. when you use `webpack -v` here webpack points to  webpack in `PATH`. `npm ...` points in your case to another location => you should have one location for both.

**Reply by hd321kbps** [reply]
The script at roaming/npm/webpack calls roaming/npm/node_modules/webpack
![image](https://user-images.githubusercontent.com/9886849/157039766-d5737428-0369-4626-a080-10a95b2f7d11.png)
![image](https://user-images.githubusercontent.com/9886849/157039886-944b94ba-2b6f-4d95-ae0a-f8fc75823d57.png)
